README.md

```diff
@@ -108,178 +115,32 @@ tuner = Tuner(
-  launches HPO locally, tuning a simple script 
-  [train_height_example.py](examples/training_scripts/height_example/train_height.py) for several baselines  
-* [launch_height_moasha.py](examples/launch_height_moasha.py):
-  shows how to tune a script reporting multiple-objectives with multiobjective Asynchronous Hyperband (MOASHA)
-* [launch_height_standalone_scheduler.py](examples/launch_height_standalone_scheduler.py):
-  launches HPO locally with a custom scheduler that cuts any trial that is not
-  in the top 80%
-* [launch_height_sagemaker_remotely.py](examples/launch_height_sagemaker_remotely.py):
-  launches the HPO loop on SageMaker rather than a local machine, trial can be executed either
-  the remote machine or distributed again as separate SageMaker training jobs. See 
-  [launch_height_sagemaker_remote_launcher.py](examples/launch_height_sagemaker_remote_launcher.py)
-  for remote launching with the help of RemoteTuner also discussed in one of the FAQs.
-* [launch_height_sagemaker.py](examples/launch_height_sagemaker.py):
-  launches HPO on SageMaker to tune a SageMaker Pytorch estimator
-* [launch_bayesopt_constrained.py](examples/launch_bayesopt_constrained.py):
-  launches Bayesian constrained hyperparameter optimization
-* [launch_height_sagemaker_custom_image.py](examples/launch_height_sagemaker_custom_image.py):
-  launches HPO on SageMaker to tune an entry point with a custom docker image
-* [launch_plot_results.py](examples/launch_plot_results.py): shows how to plot
-  results of a HPO experiment
-* [launch_tensorboard_example.py](examples/launch_tensorboard_example.py):
-  shows how results can be visualized on the fly with TensorBoard
-* [launch_nasbench201_simulated.py](examples/launch_nasbench201_simulated.py):
-  demonstrates simulation of experiments on a tabulated benchmark
-* [launch_fashionmnist.py](examples/launch_fashionmnist.py):
-  launches HPO locally tuning a multi-layer perceptron on Fashion MNIST. This
-  employs an easy-to-use benchmark convention
-* [launch_huggingface_classification.py](examples/launch_huggingface_classification.py):
-  launches HPO on SageMaker to tune a SageMaker Hugging Face estimator for sentiment classification
-* [launch_tuning_gluonts.py](examples/launch_tuning_gluonts.py):
-  launches HPO locally to tune a gluon-ts time series forecasting algorithm
-* [launch_rl_tuning.py](examples/launch_rl_tuning.py):
-  launches HPO locally to tune a RL algorithm on the cartpole environment
-* [launch_height_ray.py](examples/launch_height_ray.py):
-  launches HPO locally with [Ray Tune](https://docs.ray.io/en/master/tune/index.html)
-  scheduler
-
-## Examples for Experimentation and Benchmarking
-
-You will find many examples for experimentation and benchmarking in
-[benchmarking/examples/](benchmarking/examples/) and in
-[benchmarking/nursery/](benchmarking/nursery/).
-
-## FAQ and Tutorials
-
-You can check our [FAQ](https://syne-tune.readthedocs.io/en/latest/faq.html), to
```

Comment:
I agree that we dont want to list all items of our FAQ as there are way too many but we probably still want to mention it in the main readme.md?

Reply:
Yes, I'd put the back in a separate PR, after some revision.